src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -774,7 +774,7 @@ bool emberAfOperationalCredentialsClusterCertificateChainRequestCallback(
     }
 
     response.certificate = derBufSpan;
-    SuccessOrExit(err = commandObj->AddResponseData(commandPath, response));
+    SuccessOrExit(err = commandObj->AddResponse(commandPath, response));
```

Comment:
This is going to emberAfSendImmediateDefaultResponse in the exit block...  Again, this is actually needed for the earlier failures in this function, but we should not do it if AddResponse failed.  Followup is OK for addressing this.